gadget/gadget.go

```diff
@@ -1562,7 +1608,21 @@ func validateBareContent(vc *VolumeContent) error {
 	return nil
 }
 
-func validateFilesystemContent(vc *VolumeContent) error {
+func validateEMMCContent(name string, vc *VolumeContent) error {
```

Comment:
It is a bit strange to add the name to the content checker and then use it only in `validateEMMCContent`, while  error messages in the other checkers could benefit by printing it as well.

Reply:
I actually removed the name, inspecting the error messages from the caller chain the name is *already* included, so removed that member again